Oware/src/Player.java

```diff
@@ -2,10 +2,45 @@
 public class Player
 {
 
-	public static void main(String[] args)
+	private int playerStorehouse;
+	private int opponentStorehouse;
```

Comment:
why do we need an opponent storehouse variable here?  Let the opponent player object store its own storehouse seed count.